scripts/curl_install_pypi/install.py

```diff
@@ -41,27 +38,71 @@
     pass
 
 AZ_DISPATCH_TEMPLATE = """#!/usr/bin/env bash
-{install_dir}/{bin_dir_name}/python -m azure.cli "$@"
+{install_dir}/bin/python -m azure.cli "$@"
 """
 
-DEFAULT_INSTALL_DIR = os.path.join(os.path.sep, 'usr', 'local', 'az')
-DEFAULT_EXEC_DIR = os.path.join(os.path.sep, 'usr', 'local', 'bin')
+DEFAULT_INSTALL_DIR = os.path.expanduser(os.path.join('~', 'azure-cli'))
+DEFAULT_EXEC_DIR = os.path.expanduser(os.path.join('~', 'bin'))
 VIRTUALENV_VERSION = '15.0.0'
-BIN_DIR_NAME = 'Scripts' if platform.system() == 'Windows' else 'bin'
```

Comment:
Is this relevant if the CLI is installed with 'bash for Windows' or does it see the platform as Linux (pretty sure the latter)?